models/app-models.js

```diff
@@ -54,4 +54,23 @@ exports.selectCommentsByArticle = async (article_id) => {
     return Promise.all(querProms).then((result) => {
                 return result[querProms.length - 1].rows
             })
+}
+
+exports.insertCommentByArticle = async (article_id, comment) => {
+    let artiQuery = `INSERT INTO comments
+                        (author, body, article_id)`
+    let queryVal = [comment.username, comment.body, article_id]
+    let querProms = []
+    querProms.push(checkExists("articles","article_id",article_id))
```

Comment:
You don't need to check that the articleId exists, it is a foreign key on comments so psql will throw an error if it doesn't exist!